data/transform/pre-parser/textile-js-workarounds/fix-links.ts

```diff
@@ -8,5 +8,10 @@ export const fixDuplicateQuoteLinks: StringTransformation = (content) => content
 export const fixHtmlElementsInLinks: StringTransformation = (content) =>
   content.replace(/"([^"<]+)":([^<\s]+)</gm, '<a href="$2">$1</a><');
 
+// Punctuation immediately after links is interpreted correctly by textile-js; but was not by Nanoc.
+// We need to remove it to retain parity.
+export const fixPunctuationInLinks: StringTransformation = (content) =>
```

Comment:
In my IDE it says - TS2451: Cannot redeclare block-scoped variable 'fixPunctuationInLinks'. ?

Reply:
Sounds like an error in your IDE 🤷